### PR TITLE
Bug 1290580 - Rebase docker infra off of CentOS 6

### PR DESCRIPTION
**Pull request by dylanwh**
Alright, this should built pretty fast -- can you see if CI works?

### CODE REVIEW

File: docker/runtests.sh [hunk 1]
```diff
@@ -48,15 +48,16 @@ echo -e "\n== Starting memcached"
...
 sed -e "s?%DB_NAME%?bugs_test?g" --in-place $BUGZILLA_ROOT/qa/config/checksetup_answers.txt
 sed -e "s?%USER%?$BUGZILLA_USER?g" --in-place $BUGZILLA_ROOT/qa/config/checksetup_answers.txt
 echo "\$answer{'memcached_servers'} = 'localhost:11211';" >> $BUGZILLA_ROOT/qa/config/checksetup_answers.txt
 
 if [ "$TEST_SUITE" == "checksetup" ]; then
     cd $BUGZILLA_ROOT/qa
-    /buildbot_step "Checksetup" ./test_checksetup.pl config/config-checksetup-$BUGS_DB_DRIVER
+    /buildbot_step "Checksetup" ./checksetup.pl config/config-checksetup-$BUGS_DB_DRIVER
     exit $?
 fi
```
Comment (dklawren) [inline docker/runtests.sh]: Remove this section completely. This was an old test we no longer do that tests upgrading older databases used by older versions of Bugzilla. so test_checksetup.pl was the correct script to run, not checksetup.pl.

File: docker/bugzilla.conf [hunk 2]
```diff
@@ -11,6 +11,7 @@ ServerName localhost:80
         DirectoryIndex index.cgi
         Options Indexes FollowSymLinks ExecCGI
         AllowOverride All
-        Require all granted
+        Satisfy Any
+        Allow from all
     </Directory>
```
Comment (dklawren) [inline docker/bugzilla.conf]: We should just grab the real http conf from SCL3 webheads and use it. Minus the SSL certs, etc.

File: docker/runtests.sh [hunk 3]
```diff
@@ -75,7 +76,7 @@ perl generate_test_data.pl
 echo -e "\n== Starting web server"
 sed -e "s?^#Perl?Perl?" --in-place /etc/httpd/conf.d/bugzilla.conf
 /usr/sbin/httpd &
-sleep 3
+sleep 30
```
Comment (dklawren) [inline docker/runtests.sh]: Do we really need 30?

Comment (dylanwh) [inline docker/runtests.sh]: after a lot of testing, I think 10 is a reasonably safe value

**Pull request closed without merging**